extension-style/src/main/java/com/mapbox/maps/extension/style/sources/Source.kt

```diff
@@ -130,6 +134,17 @@ abstract class Source(
    */
   companion object {
     private const val TAG = "Mbgl-Source"
+
+    private val workerThread by lazy {
```

Comment:
Why create the workerThread here instead of GeojsonSouce?

Reply:
Yep, moved to `GeojsonSouce`